Documentation/Troubleshooting/krew-plugin.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Krew Plugin
+---
+
+## Installation
+
+* Install [Krew](https://krew.sigs.k8s.io/docs/user-guide/setup/install/)
+* Install Krew rook-ceph plugin
+  ```console
+    kubectl krew install rook-ceph
+  ```
+
+## Working
+
+Any ceph command can be run with the plugin.
+This will help in troubleshooting the cluster as by getting the stauts of the working cluster.
```

Comment:
```suggestion
## Working

Any ceph command can be run with the plugin.
This will help in troubleshooting the cluster as by getting the stauts of the working cluster.
```

removing this as there are other things from just running ceph commands

Reply:
Okay